.github/workflows/fuzzydata-test.yml

```diff
@@ -0,0 +1,33 @@
+name: fuzzy
+on:
+  pull_request:
+jobs:
+  test-fuzzydata:
+    runs-on: ubuntu-latest
+    defaults:
+      run:
+        shell: bash -l {0}
+    steps:
+      - uses: actions/checkout@v2
+        with:
+          fetch-depth: 1
+      - uses: conda-incubator/setup-miniconda@v2
+        with:
+          activate-environment: modin
+          environment-file: environment-dev.yml
+          python-version: 3.8
+          channel-priority: strict
+          use-only-tar-bz2: true # IMPORTANT: This needs to be set for caching to work properly!
+      - name: Conda environment
+        run: |
+          conda info
+          conda list
+      - uses: BSFishy/pip-action@v1
```

Comment:
In order not to install fuzzydata here, as well as for making it possible to install fuzzydata for local testing we should add it to the `environment-dev.yml` env file too.